www/docs/en/dev/config_ref/images.md

```diff
@@ -34,7 +34,7 @@ When working in the CLI you can define application icon(s) via the `<icon>` elem
 If you do not specify an icon, the Apache Cordova logo is used.
 
 ```xml
-    <icon src="res/ios/icon.png" platform="ios" width="57" height="57" density="mdpi" />
+    <icon src="res/ios/icon.png" platform="ios" width="57" height="57" />
```

Comment:
We are removing the density example here but I understand your rationale since the platorm `ios` in the example.

The `density` attribute is also missing in the table below. Maybe we should add it there. I believe `density` is optional, even for android, though it should be set if the `src` is a rasterized image (like PNG). I think `{% cdv_platform android %}` can be used to display the android icon, to indicate that it's an android only attribute. The `target` format can be followed to produce the `density` table row.

Reply:
@breautek How about changing the platform in the example to `android`?

Reply:
I reviewed https://cordova.apache.org/docs/en/12.x/config_ref/images.html

and I think that's also fine... the `density` attribute is actually documented in the android section.